src/services/room-service.spec.js

```diff
@@ -117,4 +121,37 @@ describe('room-service', () => {
     });
 
   });
+
+  describe('isRoomMemberOrOwner', () => {
+    const roomId = uniqueId.create();
+    beforeEach(async () => {
+      await roomStore.save({
+        _id: roomId,
+        name: 'my room',
+        access: ROOM_ACCESS_LEVEL.private,
+        owner: myUser._id,
+        members: [
+          {
+            userId: otherUser._id,
+            joinedOn: new Date()
+          }
+        ]
+      });
+    });
+
+    it('should return truthy when the user is the owner', async () => {
```

Comment:
I think we should check for `true` and not truthy.
To me truthy is when you don't care about the result, it just has to return something. Here we actually expect a boolean.

Reply:
I changed the code and forgot about the assertion (it used to be a truthy). Changed thanks!